layouts/about-release-schedule.hbs

```diff
@@ -14,7 +14,7 @@
         {{{ contents }}}
 
         <p>
-          <img alt="{{ title }}" src="https://raw.githubusercontent.com/nodejs/Release/master/schedule.svg?sanitize=true">
+          <img src="https://raw.githubusercontent.com/nodejs/Release/master/schedule.svg?sanitize=true" alt="{{ title }}" width="960" height="500">
```

Comment:
Should the SVG get pulled down as part of the build like the schedule.json?

Reply:
I guess it could, but this is clearly a lot easier and shouldn't really break anyway :)

I still need to fix the height auto issue.